rmb-sdk-go/examples/client_direct/main.go

```diff
@@ -3,47 +3,89 @@ package main
 import (
 	"context"
 	"fmt"
-	"os"
-	"time"
+	"math/rand"
 
+	"github.com/google/uuid"
+	"github.com/rs/zerolog/log"
 	substrate "github.com/threefoldtech/tfchain/clients/tfchain-client-go"
+	"github.com/threefoldtech/tfgrid-sdk-go/rmb-sdk-go"
 	"github.com/threefoldtech/tfgrid-sdk-go/rmb-sdk-go/direct"
+	"github.com/threefoldtech/tfgrid-sdk-go/rmb-sdk-go/direct/types"
 )
 
+type resultsChan chan bool
```

Comment:
remove the type no need if the channel is a variable you can use it from anywhere